Library/ViewModels/DeprecatedRewardPledgeViewModel.swift

```diff
@@ -497,7 +497,13 @@ public final class DeprecatedRewardPledgeViewModel: Type, Inputs, Outputs {
     )
     .map { ($0.0, $0.1, $1, $2, $3) }
     .takeWhen(Signal.merge(applePayEventAfterLogin, loggedInUserTappedApplePayButton))
-    .map(PKPaymentRequest.paymentRequest(for:reward:pledgeAmount:selectedShippingRule:merchantIdentifier:))
+    .map { PKPaymentRequest.paymentRequest(
```

Comment:
Just for curiosity, is there any reason why you decided to pass the values explicitly?

Reply:
The compiler needed help 😓